src/main/java/com/iexec/sms/authorization/AuthorizationService.java

```diff
@@ -142,6 +142,13 @@ public String getChallengeForSetAppRuntimeSecret(String appAddress,
                 Hash.sha3String(secretValue));
     }
 
+    public String getChallengeForSetAppRequesterRuntimeSecretCount(
+            String appAddress) {
+        return HashUtils.concatenateAndHash(
+                Hash.sha3String(DOMAIN),
+                appAddress);
```

Comment:
Count is missing

Reply:
Yeah, that was intentional... But falsy. Oops. :upside_down_face: 
Fixed, thanks!